### ISSUE TITLE
Testimonia 405 needs to be marked as 'syr' in the tei:ab and the tei:anchor in the edition

### ISSUE THREAD

**Issue by wlpotter** [srophe/caesarea-data#141]
Related to #132 but needs to update in tei:ab (in the `@xml:lang`) for the edition and the name of the tei:anchor (this is because #132 hadn't been implemented yet)

**Comment by wlpotter** [comment]
@davidamichelson I noticed a few other issues here.

1. There's something odd going on with the tagged placeName elements -- they look bolded to me.
2. The note formatting doesn't line up, and I just realized that I think it's because the notes are within the scope of the `ab` element, meaning they inherit the `@xml:lang="syr"`, and any formatting that goes with it? Maybe that's not true, though, since they're left-aligned. In any case, we should take a look at this one.

![Screenshot from 2022-10-17 16-23-15](https://user-images.githubusercontent.com/44096602/196286242-53b6e87a-6512-41e2-9b23-bd3ef283004a.png)
